src/test/resources/ebl/v1/shipping-instructions/basic-valid-shipping-instructions.json

```
@@ -5,8 +5,13 @@
   "isElectronic": true,
   "isShippedOnBoardType": true,
   "transportDocumentType": "SWB",
-  "freightPayableAt": {
-    "locationName": "Port of Rotterdam"
+  "invoicePayableAt": {
+    "UNLocationCode": "USNYC",

```

```suggestion
     "UNLocationCode": "DKCPH",
```

(Matching the UN Location Code with the Address)